### DISCUSSION TITLE
no formating on *.css files

### DISCUSSION THREAD

**Discussion by fabianpage** [LunarVim/LunarVim#2809] [Q&A] [answered]
Hi

I don't get any formating on *.css files. Do i need to add some config there or add a dependency?

:LspInfo:
<img width="564" alt="CleanShot 2022-07-12 at 20 51 42@2x" src="https://user-images.githubusercontent.com/8473/178572017-1f297b88-e595-4b2b-a17d-cc261290795f.png">

**Comment by fabianpage** [comment]
:LvimInfo
<img width="932" alt="CleanShot 2022-07-12 at 22 12 56@2x" src="https://user-images.githubusercontent.com/8473/178586381-097eb812-7166-467f-a54f-abd254fb1481.png">

**Comment by kylo252** [comment, marked as answer]
You can see that `cssls` doesn't have `documentFormattingProvider` as a capability (or at least it's enabled by default), so you need an external tool for that.
